### DISCUSSION TITLE
Github Desktop shownig untrusted server error

### DISCUSSION THREAD

**Discussion by raadislam** [community/community#52381] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![image](https://user-images.githubusercontent.com/110403589/230786404-a1e38c52-fd92-489a-a629-2f951e0b3b96.png)
Please Help me to solve this error

**Comment by airtower-luna** [comment]
That error message looks like *something* is messing with your HTTPS connections. Common reasons are misconfigured virus scanners or firewalls, less common (but possible) ones viruses or other attacks. I'm afraid you'll have to figure out what it is.

**Reply by raadislam** [reply]
can you suggest me  a action i should take that has probable chance of solving the issue

**Reply by airtower-luna** [reply]
No, I can't say how to solve the problem. You'll have to investigate *why* it occurs, and the *why* will answer *how* to fix it. Some points to get started:

* If you use a virus scanner, check it: Is it up to date? Did it detect any virus? Is it configured to "inspect" HTTPS connections (disable that, if it's enabled)?
* Check your firewall settings. Are there any redirect (or forward) rules that might redirect HTTPS (TCP port 443) traffic?
* Check if using another internet connection helps. E.g. if you use a laptop, connect from somewhere else, or use a VPN. If yes, something is off about the previous network (maybe a misconfigured router/firewall).

The point is to figure out what is messing with your connection.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @raadislam ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
